src/dataDisplay/table.jsx

```diff
@@ -1,21 +1,346 @@
+                     */
+                    if (definition && !basic) {
+                        const isFirstHeaderColumn = rootIndex > 0 && cellCount > 0 && stickyCells[rootIndex].className.includes('header');
+                        const definitionLeftOverflow = isFirstHeaderColumn ? 4 : 3;
+                        stickyCells[rootIndex].style.left = `${cellWidths - definitionLeftOverflow}px`;
+                    } else {
+                        // eslint-disable-next-line max-len
+                        cellWidths -= leftOverFloatSpace + (cellCount >= 3 ? auxLeftOverFloatSpace : 0);
+                        stickyCells[rootIndex].style.left = `${cellWidths}px`;
+                    }
+                }
+                if (cellCount === stickyColumnCount) {
+                    // used to get the max cell resizable width according to the table container
+                    this.setState({
+                        adjacentStickyColumnsTotalWidth: cellWidths,
+                    });
+                    cellCount = 0;
+                    cellWidths = 0;
+                }
+            }
+        }
+    }
+
+    setResizableCellsWordWrapping(shouldBreakSpaces) {
+        const {
+            stickyColumnCount,
+        } = this.props;
+        const stickyCells = this.tableRef.querySelectorAll(`.table--cell:nth-child(${stickyColumnCount})`);
+        for (let i = 0; i < stickyCells.length; i += 1) {
+            stickyCells[i].style.whiteSpace = shouldBreakSpaces ? 'break-spaces' : 'nowrap';
+        }
+        this.setStickyColumnPositions();
+    }
+
+    setStickyCellsStylesOnScroll({ scrollLeft }) {
+        const {
+            basic,
+            definition,
+            fullWidth,
+        } = this.props;
+        const stickyCellsFirstOfRow = this.tableRef.querySelectorAll(`.${STICKY_CELL_FIRST_OF_ROW_CLASS}`);
+        const stickyCellBorderLeftStyle = scrollLeft > 0 && !basic ? STICKY_CELL_BORDER_STYLE : '';
+        for (let i = 0; i < stickyCellsFirstOfRow.length; i += 1) {
+            stickyCellsFirstOfRow[i].style.borderLeft = stickyCellBorderLeftStyle;
+            const isInitalDefinitionCell = definition && !fullWidth && i === 0;
+            if (isInitalDefinitionCell) {
+                stickyCellsFirstOfRow[i].style.borderTop = backgroundColor.default;
+                stickyCellsFirstOfRow[i].style.boxShadow = '';
+                stickyCellsFirstOfRow[i].style.borderLeft = '';
+                stickyCellsFirstOfRow[i].style.borderRight = STICKY_CELL_BORDER_STYLE;
+            }
+        }
+    }
+
+    parseChildren(children) {
+        const {
+            stickyColumnCount,
+            fixed,
+            singleLine,
+            resizableColumnWidthPercentage,
+        } = this.props;
+        const {
+            adjacentStickyColumnsTotalWidth,
+            stickyTableContainerWidth,
+        } = this.state;
+        const shouldHandleStickyBehavior = !fixed && !singleLine;
```

Comment:
I tend to like some extra whitespace in places like this, but that might just be personal preference and not necessarily a team standard.

```JavaScript
const {
    foo,
    bar,
    baz,
} = this.props;

const {
    quux,
    thud,
    waldo,
} = this.state;

const xyzzy = baz || waldo;
// ...
```

Reply:
done!

Reply:
Looks fixed.  Resolving.